Landry333/Big-Owl
# #104 monitoring group mvvm

**View Changes only from this branch:** https://github.com/Landry333/Big-Owl/pull/129/files/12f958e8baf70572c3bf6d53f9d3e8563d256629..3ab146ae4c4855204408d8a9859105ac40fc27b2

-----

**Related Issues**
#104 

**Description**
The MonitoringGroupPageActivity was refactored to use MVVM style. The logic is the same, but separated to repository for db calls and ViewModel for data handling. I only made a UI change where the groups supervisor name was shown. This was unnecessary as the user monitoring the group is the user of the app.

Also MonitoringGroupListActivity was untouched as it is not used in the application anyways and will be removed in a different branch,

**How to test**
Please run the app and check that MonitoringGroup works as expected. If the user hasno group or a empty group, there shold be a pop up. If the user does have a group of users, you should be able to see the group name, the list of users and search on the users, as before
